src/Backoffice/Flight/Domain/DataTransferObject/FlightDto.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Flight\Domain\DataTransferObject;
+
+use Carbon\CarbonImmutable;
+
+class FlightDto
+{
+    public function __construct(
+        public readonly int $airlineId,
+        public readonly int $departureCityId,
+        public readonly int $arrivalCityId,
+        public readonly CarbonImmutable $departureTime,
+        public readonly CarbonImmutable $arrivalTime,
+    ) {
+    }
+}
```

Comment:
This is the same. If the class is readonly, every attribute will be too. Unless you have one or more attributes you don't want to be readonly, this is more convenient

```suggestion
readonly class FlightDto
{
    public function __construct(
        public int $airlineId,
        public int $departureCityId,
        public int $arrivalCityId,
        public CarbonImmutable $departureTime,
        public CarbonImmutable $arrivalTime,
    ) {
    }
}
```